### DISCUSSION TITLE
Feature request: new customization that, when set to `t`, strips out the new/old indicators in the display of diffs

### DISCUSSION THREAD

**Discussion by hab25** [magit/magit#5155] [Ideas]
IIUC they are redundant because the `magit-diff-...` faces already convey that information.

Advantages:
- `#'back-to-indentation` is useful again (i.e. behaves differently than `#'beginning-of-line`)
- more space in the buffer
- less distraction in the buffer

Sometimes advantageous:
- `#'kill-ring-save` never saves any output indicators. This can be better than `#'magit-copy-section-value` because, with the former, the user does not have to think about which prefix argument to use.

Note that 
`(setopt magit-diff-extra-stat-arguments '("--output-indicator-new= " "--output-indicator-old= "))`
does not work well because with it magit fails to apply the correct faces. Because I expect this to be due to a parsing error, I expect other behaviour breaks besides just the face application.

**Comment by tarsius** [comment]
I don't intend to do that but maybe #5183.

**Reply by hab25** [reply]
Thank you for your very extensive maintenance.

So that I and perhaps others reading this may better understand magit's scope and therefore better avoid opening out-of-scope requests: why not?

**Reply by tarsius** [reply]
I expect this would be a lot of work initially and then again over time more as we discover that it is causing issues in edge-cases that then have to be addressed one by one. I.e., it's a cute little (but not essential) feature, which would cost a lot of time that would better be invested elsewhere.
